WNPRC_Purchasing/src/client/components/RequestOrderPanel.tsx

```diff
@@ -60,111 +62,121 @@ export const RequestOrderPanel: FC<Props> = memo(props => {
-                    </Col>
-                </Row>
-                <Row>
-                    <Col xs={11} lg={6}>
-                        <BusinessPurposeInput
-                            value={model.justification}
-                            hasError={model.errors?.find(field => field.fieldName === 'justification')}
-                            onChange={onValueChange}
-                        />
-                    </Col>
-                    <Col xs={11} lg={6}>
-                        <SpecialInstructionInput value={model.comments} onChange={onValueChange} />
-                    </Col>
-                </Row>
-                {/* <AccountInput*/}
-                {/*    value={model.account}*/}
-                {/*    hasError={model.errors?.find((field) => field.fieldName === 'account')}*/}
-                {/*    onChange={onValueChange}*/}
-                {/* />*/}
-                {/* {*/}
-                {/*    (showOtherAcct || model.account === "Other") &&*/}
-                {/*    <AccountOtherInput*/}
-                {/*            value={model.otherAcctAndInves}*/}
-                {/*            hasError={model.errors?.find((field) => field.fieldName === 'otherAcctAndInves')}*/}
-                {/*            hasOtherAcctWarning={!!model.otherAcctAndInvesWarning}*/}
-                {/*            onChange={onValueChange}*/}
-                {/*    />*/}
-                {/* }*/}
-                {/* <VendorInput*/}
-                {/*    hasError={model.errors?.find((field) => field.fieldName === 'vendorId')}*/}
-                {/*    onChange={onValueChange}*/}
-                {/*    model={model}*/}
-                {/*    onModelChange={onModelChange}*/}
-                {/* />*/}
-                {/* <BusinessPurposeInput*/}
-                {/*    value={model.justification}*/}
-                {/*    hasError={model.errors?.find((field) => field.fieldName === 'justification')}*/}
-                {/*    onChange={onValueChange}*/}
-                {/* />*/}
-                {/* <SpecialInstructionInput*/}
-                {/*    value={model.comments}*/}
-                {/*    onChange={onValueChange}*/}
-                {/* />*/}
-                {/* <ShippingDestinationInput*/}
-                {/*    value={model.shippingInfoId}*/}
-                {/*    hasError={model.errors?.find((field) => field.fieldName === 'shippingInfoId')}*/}
-                {/*    onChange={onValueChange}*/}
-                {/* />*/}
-                {/* <DeliveryAttentionInput*/}
-                {/*    value={model.shippingAttentionTo}*/}
-                {/*    hasError={model.errors?.find((field) => field.fieldName === 'shippingAttentionTo')}*/}
-                {/*    onChange={onValueChange}*/}
-                {/* />*/}
-            </Form>
-            {model.errorMsg && <div className="alert alert-danger">{model.errorMsg}</div>}
-        </Panel>
+        <>
+        {
+            !isAdmin && canUpdate && (
```

Comment:
Generally put these two on the same line for readability

Reply:
ok, done